src/app/legacy/psammead/psammead-story-promo/src/index.jsx

```diff
@@ -105,27 +95,23 @@ const ImageOverlayWrapper = styled.div`
   ${({ promoType }) => mediaIndicatorStyles[promoType]}
 `;
 
-const headlineTopStoryTypography = script => getParagon(script);
-
-const headlineRegularTypography = script => getPica(script);
-
-const headlineLeadingStoryTypography = script => getDoublePica(script);
-
-const headlineTypography = script => ({
-  top: headlineTopStoryTypography(script),
-  regular: headlineRegularTypography(script),
-  leading: headlineLeadingStoryTypography(script),
+// TODO: verify
```

Comment:
This component is no longer used, apart from `Link` (line 165). 

We won't be able to / need to verify this, so we should remove as much of this code as possible.